src/config/config.js

```diff
@@ -1,17 +1,31 @@
+// import Message from "./message.js";
+// import Event from "./event.js";
+import { eventsArray } from "./event.js";
+
 export default class Config {
-  constructor(frequency = 5000, endHour = 23, endMinute = 55) {
-      this.frequency = frequency;
-      this.endHour = endHour;
-      this.endMinute = endMinute;
+  static BuddyProfilePicSrc = "./assets/images/buddy-profile-pic-cat.png";
+
+  constructor(frequency = 5000, endHour = 23, endMinute = 59, userName = "buddy") {
+    this.frequency = frequency;
+    this.endHour = endHour;
+    this.endMinute = endMinute;
+    this.eventsArray = eventsArray;
```

Comment:
```suggestion
    this.events = events;
```
Plural should indicate that it is a list